packages/clerk-js/src/ui/signUp/SignUpContinue.tsx

```diff
@@ -100,24 +100,15 @@ function _SignUpContinue(): JSX.Element | null {
       setError(undefined);
       const req = buildRequest(fieldsToSubmit);
       const res = await signUp.update(req);
-      return completeSignUpFlow(res);
-    } catch (err) {
-      handleError(err, fieldsToSubmit, setError);
-    }
-  };
 
-  const completeSignUpFlow = (su: SignUpResource) => {
-    if (su.status === 'complete') {
-      return setActive({
-        session: su.createdSessionId,
-        beforeEmit: navigateAfterSignUp,
+      return CompleteSignUpFlow({
+        signUp: res,
+        signUpUrl: displayConfig.signUpUrl,
+        handleComplete: () => setActive({ session: res.createdSessionId, beforeEmit: navigateAfterSignUp }),
```

Comment:
🙃 Let's keep useSession for now. We will change it to setActive in a separate PR. Is this correct @igneel64 and @alex-ntousias ?

Reply:
Yeap, as discussed yesterday, we are moving `setActive` to experimental currently. We will be changing it to `setSession`.

Reply:
Restored back to setSession instead of setActive e7c2ed56b2edb6fc49fd121a4345e644582fab9d